libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -415,6 +415,14 @@ public MapboxNavigation retrieveMapboxNavigation() {
     return navigationViewModel.retrieveNavigation();
   }
 
+  public void initializeOfflineData(String tileFilePath, String translationsDirPath) {
```

Comment:
Let's add javadoc for these new methods here - I added some javadoc in `nav-native-offline` that you can take a look at after rebasing.